samples/tab-user-profile/ts/README.md

```diff
@@ -0,0 +1,103 @@
+---
+page_type: sample
+description: Sample tab-user-profile app.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- typescript
+extensions:
+ contentType: samples
+ createdDate: "20/06/2023 01:38:27 PM"
+urlFragment: officedev-microsoft-teams-samples-tab-user-profile-ts
+---
+
+# profile
+
+Opens a profile card at a specified position to show profile information about a persona.
+
+ ## Included Features
+* Tabs
+
+## Interaction with app
+
+![meta-OS-profileGif](Images/meta-OS-profileGif.gif)
+
+## Prerequisites
+-  [NodeJS](https://nodejs.org/en/)
+
+-  [M365 developer account](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) or access to a Teams account with the appropriate permissions to install an app.
+- [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
+  
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## Setup
+    
+1. Setup NGROK
+  - Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http 3978 --host-header="localhost:3978"
+    ```
+2. Setup for code
+ - Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+-  In a terminal, navigate to `samples/tab-user-profile/ts`
+
+ - Install modules
+
+    ```bash
+    npm install
+    ```
+ - Run your app
+
+    ```bash
+    npm start
+    ```
+3. Setup Manifest for Teams
+
+- **This step is specific to Teams.**
+    **Edit** the `manifest.json` contained in the `Manifest/` folder to replace with your Manifest-Id *everywhere* you see the place holder string `{{Manifest-Id}}`.
+    **Edit** the `manifest.json` for `validDomains` and replace `{{domain-name}}` with base Url of your domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app`.
```

Comment:
This place holder does not exist in your manifest: {{domain-name}}`
Kindly add it.

Reply:
Fixed